### ISSUE TITLE
Adding the trained the model in the repository

### ISSUE THREAD

**Issue by VijayV28** [ajaykumarvarma/doctor_prescription_recognization_using_dl#1]
Hey,

Given the free resources of Google Colab, training the model for 50 epochs doesn't work well with my computer. I even tried to train the model with 10 epochs at once but my loss doesn't seem to improve much. It would be really helpful if you could add the trained model to your repository or provide access to it in any other way. Thank you.

Vijay

**Comment by ajaykumarvarma** [comment]
@VijayV28 i will do it. At present iam having exams. After finishing my exams i will do it. BI-lstm gives 81% accuracy,

**Comment by ajaykumarvarma** [comment]
@VijayV28 i forgot to say one thing. There is custom layer called CTC Layer. We can save model if we are having custom layers. At that time i dont have option to make to saved into h5 file.

do one thing just share screenshot of execution or accuracy part

**Comment by VijayV28** [comment]
Sure thing. please do check the screenshots attached below:

![image](https://user-images.githubusercontent.com/81868201/234947732-5b1af77c-f964-4d52-b994-d038bf2c2f22.png)

I can also share the complete notebook if you can provide your mail ID.

**Comment by VijayV28** [comment]
> @VijayV28 i will do it. At present iam having exams. After finishing my exams i will do it. BI-lstm gives 81% accuracy,

Thank you very much

**Comment by ajaykumarvarma** [comment]
@VijayV28 my email id - ajaynagaraju32@gmail.com

**Comment by ajaykumarvarma** [comment]
@VijayV28 i have checked your code. If you use same network architecture which i used, you may get varying accuracy in between 79-82.  Try changing layers (neural network topology) you can get varied accuracy.

**Comment by ajaykumarvarma** [comment]
@VijayV28 if u know about working of conv layers, just think once, ur input image and how many poolings to add and as you know pooling makes image (feature matrix) to decrease the size ( example : 3*3 -> 2*2 ) .

based on this once design your architecture by viewing research papers. What network topology they used.

you can surely get good topology which gives good accuracy.

**Comment by VijayV28** [comment]
This was really helpful. Thank you very much

**Comment by HaniaGhouse0407** [comment]
@VijayV28 hello, Were you able to get better accuracy?
@ajaykumarvarma hello, I am trying to recreate this model, but the highest accuracy I am getting even after tinkering with the layers is 78%. Will I get better accuracy by changing the decoder method? I want to change the decoder method to word beam search or prefix beam search, but I am having difficulty doing so.
 I am trying to implement this word decoder: https://github.com/githubharald/CTCWordBeamSearch

**Comment by ajaykumarvarma** [comment]
@HaniaGhouse0407 once try with resnet type of architecture for CNN layers. Instead of going with forward CNN layers use some skip connections and conv blocks (Resnet). This might help to better learn features. The final thing to do is to use a vision transformer as the training batch varies the training might get varied so better use attention mechanism.

**Comment by VijayV28** [comment]
Hello @HaniaGhouse0407, tinkering with the existing layers did not show any significant improvement in performance for me. Try using ResNet as suggested by @ajaykumarvarma.
